### DISCUSSION TITLE
Помогите столкнулся с проблемой не работал дискорд, решил снести goodbyedpi и видимо криво снес столкнулся с проблемой

### DISCUSSION THREAD

**Discussion by edas1421** [Flowseal/zapret-discord-youtube#492] [Q&A]
![Desktop Screenshot 2024 10 31 - 01 26 50 48](https://github.com/user-attachments/assets/61bcb43d-660e-4548-812f-156102c1c86a)
![Desktop Screenshot 2024 10 31 - 01 26 28 29](https://github.com/user-attachments/assets/3046c40d-e130-4c3a-ba22-6f16405ffadd)

**Comment by edas1421** [comment]
Сначала отключал в службах ибо папка bin не сносилась, потом в ручную в службах отключал и видимо натыкал, знатоки подскажите как исправить, в служабах в ручную не запускается заново.

**Reply by edas1421** [reply]
> Сначала отключал в службах ибо папка bin не сносилась, потом в ручную в службах отключал и видимо натыкал, знатоки подскажите как исправить, в служабах в ручную не запускается заново.

winws так же не запускается

**Reply by edas1421** [reply]
> > Сначала отключал в службах ибо папка bin не сносилась, потом в ручную в службах отключал и видимо натыкал, знатоки подскажите как исправить, в служабах в ручную не запускается заново.
> 
> winws так же не запускается

WinDivert сносил в  ручную тоже.

**Comment by edas1421** [comment]
![Desktop Screenshot 2024 10 31 - 01 44 25 16](https://github.com/user-attachments/assets/979a50f7-e776-4414-b469-94b8ddf81a87)
Так же при попытки снести еще раз, видимо криво удалил WinDivert и все пошло по одному месту

**Reply by eddmzh** [reply]
решил проблему?
